data_collection/web/guides.rst

```diff
@@ -221,187 +221,184 @@ indicate no value.
-        1                           // quantity
-    ]);
-
-    // track cart update with a total value of 6.00
-    _paq.push(["trackEcommerceCartUpdate", 6.00]);
-
-This code snippet sends a cart update event with a cart containing one item (SKU *candy-12837*, name *MEGA Milk Chocolate 200g*, category *Candy*, price *6.00*) and having total value of *6.00*.
-
-The list of registered items is stored only in memory. **Reloading the page will clear the list** and the previously registered items will have to be added again.
-
-.. code-block:: javascript
-
-    // visitor added one mango fruit to a shopping cart with one chocolate bar
-
-    // register previously added items
-    _paq.push(["addEcommerceItem", "82775027", "MEGA Milk Chocolate 200g", "Candy", 6.00, 1]);
-
-    // register the new item
-    _paq.push(["addEcommerceItem", "01809926", "FRUTASTIC Mango", "Fruits & vegetables", 4.00, 1]);
-
-    // track cart update with a total value of 10.00
-    _paq.push(["trackEcommerceCartUpdate", 10.00]);
-
-.. note::
-
-    If you are not sure what items have been registered, use :ref:`getEcommerceCart<jtc-api-getEcommerceItems>` function.
-
-    .. code-block:: javascript
-
-        _paq.push([function() { console.log(this.getEcommerceItems()); }]);
+Another type of e-commerce activity you can track is an update of a shopping cart.
+With it, we are able to measure how often visitors add or remove specific products to the cart and what products stay in abandoned carts.
 
-Because single page applications do not refresh the page when a visitor manipulates the cart, an e-commerce implementation in SPAs must either:
-
-1. Clear the cart using :ref:`clearEcommerceCart<jtc-api-clearEcommerceCart>` and register all items from the cart before tracking cart update, e.g.
-
-.. code-block:: javascript
+Tracking a cart update has two steps:
 
-    // visitor added one chocolate bar to an empty shopping cart
-    _paq.push(["clearEcommerceCart"]);
-    _paq.push(["addEcommerceItem", "82775027", "MEGA Milk Chocolate 200g", "Candy", 6.00, 1]);
-    _paq.push(["trackEcommerceCartUpdate", 6.00]);
+  * cart update (:ref:`ecommerceCartUpdate<jtc-api-ecommerceCartUpdate>`)
+  * product addition and removal (:ref:`ecommerceAddToCart<jtc-api-ecommerceAddToCart>` and :ref:`ecommerceRemoveFromCart<jtc-api-ecommerceRemoveFromCart>`)
 
-    // visitor added one mango fruit to a shopping cart with one chocolate bar
-    _paq.push(["clearEcommerceCart"]);
-    _paq.push(["addEcommerceItem", "82775027", "MEGA Milk Chocolate 200g", "Candy", 6.00, 1]);
-    _paq.push(["addEcommerceItem", "01809926", "FRUTASTIC Mango", "Fruits & vegetables", 4.00, 1]);
-    _paq.push(["trackEcommerceCartUpdate", 10.00]);
+Cart update allows us to synchronize longer living cart session (containing previously selected products) with short living visitor session.
+This command is optional but recommended. It should be used once per page (immidiately after loading).
```

Comment:
`immediately` instead of `immidiately`

Reply:
I think we should also add some articles here.
`A cart update allows us to synchronize a longer living cart session [...] with a shorter living visitor session.`